### DISCUSSION TITLE
Sunbird lern discussion forum local installation

### DISCUSSION THREAD

**Discussion by nagendraprasady** [Sunbird-Lern/Community#102] [Installation] [answered]
https://lern.sunbird.org/use/developer-installation/discussion-forum/installation-guide/nodebb-setup
reference this url  try do local setup getting this error I am using this command:  ./nodebb setup
<img width="941" alt="image" src="https://user-images.githubusercontent.com/128679814/231377454-4e5bd787-eb11-414d-96af-7a78a4aad75a.png">

**Comment by reshmi-nair** [comment, marked as answer]
@nagendraprasady We will check this and get back

**Comment by reshmi-nair** [comment]
@nagendraprasady Currently we are using Ubuntu to setup discussion forum. I saw from the screenshot that you are trying to use windows. Can you please check this link from NodeBB community itself for this? [NodeBB on Windows](https://docs.nodebb.org/installing/os/windows/). Please clarify the installation queries with NodeBB team. 

Also kindly contribute to us on how to configure in Windows once you have completed the setup. Thank you.
